libs/libcommon/include/common/unaligned.h

```diff
@@ -16,12 +16,14 @@
 
 #include <string.h>
 
+#include <bit>
 #include <type_traits>
 
 
 template <typename T>
 inline T unalignedLoad(const void * address)
 {
+    static_assert(std::endian::native == std::endian::little, "Only little endian is supported");
```

Comment:
`static_assert` only ensure the compile machine is little endian.
Maybe we need to add a explicit check when tiflash starts. And do that in a standalone PR

Reply:
reverted